library/src/G4CMPSolidUtils.cc

```diff
@@ -261,7 +263,8 @@ void G4CMPSolidUtils::AdjustToClosestSurfacePoint(G4ThreeVector& pos,
 // Get edge position along vTan from pos
 G4ThreeVector G4CMPSolidUtils::GetEdgePosition(const G4ThreeVector& vTan,
                                                const G4ThreeVector& pos,
-                                               G4double high, const G4int curvedSurf) const {
+                                               G4double high,
+                                               const G4int curvedSurf) const {
```

Comment:
I'm curious about why you broke this into two lines -- did it go past 80 columns?

Also, it looks like line 266 might have extra spaces at the end of the line; those should be removed.